### DISCUSSION TITLE
API URL

### DISCUSSION THREAD

**Discussion by thientrinhcoder** [QilinCloud/QilinCloud#3] [General]
As discussion in PBI https://dev.azure.com/marcossoftware/Qilin/_workitems/edit/4216

The API URL should have the schema :
https://<stage?>api.qilin.cloud/<apiName>-management/<version>/<endpoint>

API URL should follow the Restfull API convention
https://restfulapi.net/resource-naming/

For example:

Test environment:
https://test.api.qilin.cloud/product-management/v1.0/products/
https://test.api.qilin.cloud/product-management/v1.0/brands/

Dev enviroment
https://dev.api.qilin.cloud/offer-management/v1.0/warehouses/

Production environment
https://api.qilin.cloud/order-management/v1.0/orders/

**Comment by marc-costea** [comment]
👍
